webui/react/src/hooks/usePermissions.ts

```diff
@@ -230,6 +232,20 @@ const canViewGroups = ({ rbacReadPermission, rbacEnabled, user }: RbacOptsProps)
   return rbacReadPermission || (!!user && (rbacEnabled || user.isAdmin));
 };
 
+const canViewModelRegistry = ({
+  rbacReadPermission,
+  rbacEnabled,
+  user,
+  userAssignments,
+  userRoles,
+}: RbacOptsProps): boolean => {
+  const permitted = relevantPermissions(userAssignments, userRoles);
+  return (
+    rbacReadPermission ||
+    (!!user && (rbacEnabled ? permitted.has(V1PermissionType.VIEWMODELREGISTRY) : user.isAdmin))
```

Comment:
Is it sufficient?

Reply:
@keita-determined as written isd `rbac` is not enabled, then only `administrators` can view the model registry. Is that what we want?

Reply:
Thats a good question. I had the same question. Let me ask in slack channel

Reply:
@julian-determined-ai 
Everyone can see model registry tab in workspace and global model registry for OSS
changed the code